gcs/src/it/scala/org/broadinstitute/monster/storage/gcs/GcsApiIntegrationSpec.scala

```diff
@@ -30,13 +29,19 @@ class GcsApiIntegrationSpec
 
   private val bucket = "broad-dsp-monster-dev-integration-test-data"
 
-  /*
-   * NOTE: This is lorem ipsum because it needs to be large enough to meaningfully test
-   * GCS's range downloads, and I was too lazy to come up with my own paragraph of text.
-   */
-  private val bodyText = Random.alphanumeric.take((2.5 * GcsApi.ChunkSize).toInt).mkString
+  private def bodyText(n: Int): Stream[IO, Byte] = {
+    Stream
+      .randomSeeded(n.toLong)[IO]
+      .map(String.valueOf)
+      .flatMap(s => Stream.emits(s.getBytes))
+      .take(n.toLong)
+  }
 
-  private val bodyMd5 = DigestUtils.md5Hex(bodyText)
+  // smaller than MaxBytesPerUploadRequest
+  private val bodySize = 2 * GcsApi.ChunkSize
+  private val body = bodyText(2 * GcsApi.ChunkSize)
+  // TODO: best way to get Md5
+  private val bodyMd5 = DigestUtils.md5Hex(buildString(body).unsafeRunSync().toString())
```

Comment:
I think the top-level `body` and `bodyMd5` should go away, so tests are always forced to be explicit about how they're generating the tested text.